config/config_comgsi.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Compiler/MPI combination
+export HPC_COMPILER=${HPC_COMPILER:-"intel/18.0.5"}
```

Comment:
What machine does this config refer to?

Reply:
That value is for Cheyenne but these three HPC_* variables are just place holders.  Community users needs to change them or set corresponding environments for hpc-stack to work (community users spread around the world, not limited to NOAA and NCAR machines). Thanks!

Reply:
Do we need this specific config then? I'm fine with the `stack_comgsi`, but I'm not sure this config adds anything.

Reply:
I would hope to add them. But I am okay if it is decided to not include them and if so, community users will need to get a copy from the comgsi/hpc-stack fork (not as official as from NOAA-EMC/hpc-stack).

Reply:
Maybe a different name then? It's not clear to me that this is meant to be used on Cheyenne, and none of the other settings are unique. You can't just run use this config on any machine. Or is GSI development usually done on Cheyenne?

Reply:
Do you have any suggestions on "a different name"?  Yes, I mainly develop and test community GSI on Cheyenne.

Reply:
How about `config_Cheyenne_GSI?`

Reply:
Ah, I think that may not be appropriate. These two files are NOT used on Cheyenne solely.  I also used it on Orion to build libraries for regional GSI/EnKF ( I only need to export HPC_COMPILER and HPC_MPI variables). Users also use them to build their comgsi-flavor hpc-stack, for example on Stampede of TACC, on supercomputers at CSU, etc. I got what you mean. You'd like to see this file is specific to one particular machine. But my intention is to get ready a file specifically customized for building community GSI (mostly for regional applications). If this goal conflicts with some hpc-stack policies, it is fine to not include these two files. Thanks!

Reply:
I'm convinced with your explanation.

Reply:
Thanks!